.github/ISSUE_TEMPLATE/new_package.yml

```diff
@@ -0,0 +1,75 @@
+  - type: input
+    id: name
+    attributes:
+      label: Tool Name
+      description: The name of the tool being installed, normally different from the package name. For example fakenet-ng.vm (package name) vs FakeNet-NG (tool name).
+      placeholder: ex. FakeNet-NG
+    validations:
+      required: true
+  - type: dropdown
+    id: browsers
+    validations:
+      required: true
+    attributes:
+      label: Category
+      description: Which category should this tool be installed to?
+      options:
+        - Android
+        - Debuggers
+        - Delphi
+        - Developer Tools
+        - Disassemblers
+        - dotNet
+        - Flash
+        - Forensic
+        - Hex Editors
+        - Java
+        - Javascript
+        - Net
+        - Office
+        - PDF
+        - Pentest
+        - PowerShell
+        - Python
+        - Text Editors
+        - Utilities
+        - VB
+        - Web Application
+  - type: input
+    id: url
+    attributes:
+      label: Download URL
+      description: URL to download the tool installer. Use a URL that is not likely to be deleted/unavailable and whose downloaded file is not likely to change as this will eventually break the package. GitHub releases links are preferred as they allow us to automatically update the package to use a newer version.
+      placeholder: ex. https://github.com/mandiant/flare-fakenet-ng/releases/download/v1.4.11/fakenet1.4.11.zip
+    validations:
+      required: true
+  - type: input
+    id: hash
+    attributes:
+      label: Download SHA256 Hash
+      description: The SHA256 Hash of the downloaded installer is used for verification.
+      placeholder: ex. 62af5cce80dbbf5cdf961ec9515549334a2112056d4168fced75c892c24baa95
+  - type: textarea
+    id: why
+    attributes:
+      label: Why is this tool a good addition?
+      description: Tell us why you would like to have this package in FLARE-VM and how this tool is useful for malware analysis, incident response, penetration testing and other security related tasking. Adding new packages involves more maintenance work, tell us what is unique about this tool that can not be done with the current toolset.
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: |
+        ---
+        We really appreciate that you share your package idea with us and we will review you issue as soon as possible. But please understand that our time is limited and we may not be able to create a package for this tool as fast as you may like. If you would like to speed the process up, we encourage you to check our [Contributing Wiki page](https://github.com/mandiant/VM-Packages/wiki/Contributing) and send a pull request with the new package. We would really like that the community help us creating new packages and maintaining the existing ones. Join us! :hugs:
```

Comment:
Any feedback on this @mandiant/flare-vm? I want to encourage people to contribute and send package ideas, but also make clear that sending an issue with a package proposal doesn't mean we are going to migrate it. Realistically we may not have time to create all packages, I want to avoid people getting disappointed and knowing in advance that PR are preferred over only an issue.

Reply:
some suggestions on phrasing but overall good message!